packages/components/src/tabs/styles.ts

```diff
@@ -34,24 +34,46 @@ export const TabListWrapper = styled.div`
 		position: absolute;
 		pointer-events: none;
 
+		--direction-factor: 1;
+		--indicator-start: var( --indicator-left );
+		&:dir( rtl ) {
```

Comment:
Due to the influence of c347fad, this selector is now applied to the `::after` pseudo-element. As I understand it, we cannot use `:dir()` on the pseudo-element itself. As a result, in RTL languages, `--direction-factor` remains `1` instead of `-1`, and the indicator is not displayed 😅

![direction-factor](https://github.com/user-attachments/assets/21e07f18-e95a-412d-841b-94d6c1959397)

Reply:
Good catch lol. Fixed!